solution/ui/regulations/css/scss/partials/_search.scss

```diff
@@ -9,6 +9,75 @@
     margin-bottom: 1em;
 }
 
+.search-field {
+    background-color: #fff;
```

Comment:
these styles were removed from `SearchInput.vue` and put here in the stylesheet, which is a more appropriate place.  We're moving away from using `<style>` tags in Single File Components.